Formula/sevenzip.rb

```diff
@@ -0,0 +1,24 @@
+class Sevenzip < Formula
+  desc "7-Zip is a file archiver with a high compression ratio"
+  homepage "https://7-zip.org"
+  url "https://7-zip.org/a/7z2102-src.7z"
+  version "21.02a"
+  sha256 "ee9755feaa034e7075712bc6b1fad25440bd0a8f21f106a16e37bc4b409f1122"
+  license all_of: ["LGPL-2.1-or-later", "BSD-3-Clause"]
+
+  def install
+    cd "CPP/7zip/Bundles/Alone2" do
+      suffix = Hardware::CPU.arm? ? "arm64" : "x64"
```

Comment:
I believe this is the same
```suggestion
      suffix = Hardware::CPU.arch
```